photon-client/src/components/settings/NetworkingCard.vue

```diff
@@ -38,9 +40,31 @@ const isValidHostname = (v: string | undefined) => {
   return hostnameRegex.test(v);
 };
 
+const settingsHaveChanged = (): boolean => {
+  const a = useSettingsStore().network;
+  const b = tempSettingsStruct.value;
+
+  return (
```

Comment:
Why can't we use === here?

Reply:
Object equality checks are very bad in js and is a likely risk, better to just do this. Its the same way we compare resolutions manually in the frontend

Reply:
💀